test/__snapshots__/storyshots.test.js.snap

```diff
@@ -7272,6 +7281,7 @@ exports[`RippleStoryshots Molecules/Form Form 1`] = `
                 class="form-control"
                 id="text-area"
                 placeholder="Start typing..."
+                required="required"
```

Comment:
Not sure why - but this reverts even on master branch (before these changes are implemented).

Reply:
I guess it's from vfg update.